lib/ControlledVocab/ControlledVocab.js

```diff
@@ -132,6 +136,23 @@ class ControlledVocab extends React.Component {
       connect: PropTypes.func.isRequired,
       hasPerm: PropTypes.func.isRequired
     }).isRequired,
+    /*
+      sample translation prop
+      {
+        cannotDeleteTermHeader: "Cannot delete patron group",
+        cannotDeleteTermMessage: "This patron group cannot be deleted, as it is in use by one or more records.",
+        deleteEntry: "Delete patron group",
+        termDeleted: "The patron group <b>{term}</b> was successfully <b>deleted</b>",
+        termWillBeDeleted: "The patron group <b>{term}</b> will be <b>deleted.</b>"
+      }
+    */
```

Comment:
I expect documentation in the readme.md following the examples in https://github.com/folio-org/stripes-smart-components/tree/master/lib/ConfigManager and https://github.com/folio-org/stripes-components#links-to-documentation-of-specific-components-and-utilities

Reply:
Agreed, per-component README files would be great, @julianladisch. If we had such documentation already, then it would be easy to add this new prop to it, and we'd be done! 

But given there is no documentation whatsoever, let's handle that task in a separate ticket so we don't explode the scope and point estimate of this ticket, which are already accepted. Then, we can also assign that task to somebody who is familiar with the full scope of this component.